charts/zwavejs2mqtt/values.yaml

```
@@ -0,0 +1,104 @@
+# Default values for zwavejs2mqtt.
+
+image:
+  repository: zwavejs/zwavejs2mqtt
+  pullPolicy: IfNotPresent
+  tag: 1.0.0-alpha.2
+
+strategy:
+  type: Recreate
+
+# # See more environment varaibles in the zwavejs2mqtt documentation
+# https://zwave-js.github.io/zwavejs2mqtt/#/guide/env-vars
+env:
```

```suggestion
env: {}
```
Might be cleaner to set `env` to an empty dict

---

damn yea. it would have errored out otherwise